sdk/edge/Azure.ResourceManager.EdgeConnectivityStatuses/src/autorest.md

```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: Edge
+namespace: Azure.ResourceManager.Edge
+require: https://raw.githubusercontent.com/roopavidhya/azure-rest-api-specs-pr/45dce3521d347333f5f41e21d5763f192abc4c79/specification/edge/resource-manager/Microsoft.Edge/connectivityStatuses/readme.md
```

Comment:
@roopavidhya As mentioned in previous PR, we will consider reviewing the PR after the swagger is merged into public main.
And also, have you got the namespace approved for your service?

Reply:
We have merged it to RPSaaSMaster. Is it also necessary to merge it into azure-rest-api-specs-pr/main?

Reply:
Regarding the namespace, yes. The swagger is ARMSignedOff

Reply:
> We have merged it to RPSaaSMaster. Is it also necessary to merge it into azure-rest-api-specs-pr/main?

The swagger needs to be merged into azure-rest-api-specs/main, not private spec repo.

Reply:
> Regarding the namespace, yes. The swagger is ARMSignedOff

Namespace approval is something like [this](https://github.com/Azure/azure-sdk-pr/issues/1220)

Reply:
Sorry this link isn't accessible for me 
> 
> Namespace approval is something like [this](https://github.com/Azure/azure-sdk-pr/issues/1220)

Reply:
> Sorry this link isn't accessible for me
> 
> > Namespace approval is something like [this](https://github.com/Azure/azure-sdk-pr/issues/1220)

You can raise the request in the public repo, like [this](https://github.com/Azure/azure-sdk/issues/6473)